src/static/thankyou.html

```diff
@@ -40,6 +40,7 @@
     <div class="content-wrapper">
        <div class="order-form-wrapper">
             <h1>Thank you for your order!</h1>
+            <h1>Your order of </h1>
```

Comment:
this is another page that can be deleted and replaced with the thankyou.php page